### PR TITLE
feat: custom menu

### PR DESCRIPTION
**Pull request by pgolovkin**
@stepanLav this PR has to be tested on Windows, Mac, Linux.
`Nova Spektr` should be in the menu but not the `nova-spektr`.

### CODE REVIEW

File: src/main/vectormenu.ts [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+          label: 'Zoom Out',
+        },
+        {
+          role: 'togglefullscreen',
+          label: 'Toggle Full Screen',
+        },
+        {
+          role: 'toggleDevTools',
+          label: 'Toggle Developer Tools',
+        },
```
Comment (Asmadek) [inline src/main/vectormenu.ts:85]: Maybe better to hide dev tools for prod version?

File: src/main/vectormenu.ts [hunk 2]
```diff
@@ -0,0 +1,188 @@
+import { app, shell, Menu, MenuItem, MenuItemConstructorOptions } from 'electron';
+
+import { ENVIRONMENT, PLATFORM } from '@shared/constants';
+
+export function buildMenuTemplate(): Menu {
+  const template: Array<MenuItemConstructorOptions | MenuItem> = [
```
Comment (tuul-wq) [inline src/main/vectormenu.ts:6]:
```suggestion
  const Template: MenuItemConstructorOptions[] | MenuItem[] = [
```

**Review by tuul-wq — APPROVED** [review]
Nice 👍

File: src/main/vectormenu.ts (hunk 2 above)
Comment (pgolovkin) [reply src/main/vectormenu.ts:6]: Agree with the type, but why the constant name starts from the capital `T`?

**Review by Asmadek — APPROVED** [review]

**Pull request merged**